modules/instrument_builder/jsx/react.instrument_builder.js

```diff
@@ -325,7 +329,7 @@ class DisplayElements extends Component {
         );
       }
       return row;
-    });
+    }.bind(this));
```

Comment:
i agree with @HenriRabalais. i don't think this is the way to fix the bug

Reply:
You and @HenriRabalais are correct! 
I also found a blog post about it: https://ryanfunduk.com/articles/never-bind-in-render/
New changes remove it from the render function.